stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java

```diff
@@ -183,15 +182,20 @@ public void onStartTrackingTouch(SeekBar seekBar) { }
       @Override
       public void onStopTrackingTouch(SeekBar seekBar) { }
     });
+
     mPercentClipMinSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
       @Override
       public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
         mPercentClipMin = progress;
         updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
-        // move max to march min if max goes below min
-        if (mPercentClipMax < mPercentClipMin) {
-          mPercentClipMax = mPercentClipMin;
-          updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
+        if (mPercentClipMin + mPercentClipMax > 100) {
+            // constrain min + max <= 100
+            mPercentClipMax = 100 - mPercentClipMin;
+            updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
+        } else if (mPercentClipMax < mPercentClipMin){
```

Comment:
min and max define the percentile cutoff below and above respectively. Max is not necessarily > min. we should not put max > min as a constrain.